src/lib/y2partitioner/widgets/configure.rb

```diff
@@ -89,9 +89,17 @@ def help
 
       # @return [Array<Yast::WidgetTerm>]
       def items
-        actions.map do |action|
-          Item(Id(action.id), Yast::Term.new(:icon, action.icon), action.label)
-        end
+        actions.map { |a| item(a) }
+      end
+
+      # Generates an item for the given action
+      #
+      # @param action [Action]
+      # @return [Yast::WidgetTerm]
+      def item(action)
+        return Item(Id(action.id), action.label) unless action.icon?
+
+        Item(Id(action.id), Yast::Term.new(:icon, action.icon), action.label)
```

Comment:
Just a doubt, why to keep this?

In the PR description you wrote

> For now, **all icons have been dropped** from the Configure menu button.

Reply:
Well, maybe in the future we find good icons for those actions, so I have extended the `Action` class to allow both: with and without icon. This will make easier to add icons then.